velox/dwio/common/Options.h

```diff
@@ -580,6 +580,7 @@ struct WriterOptions {
   const velox::common::SpillConfig* spillConfig{nullptr};
   tsan_atomic<bool>* nonReclaimableSection{nullptr};
   std::optional<velox::common::CompressionKind> compressionKind;
+  std::shared_ptr<dwio::common::FlushPolicy> flushPolicy;
```

Comment:
Why not make this a FlushPolicyFactory? It is odd to pass a FlushPolicy here and make it a FlushPolicyFactory later.

Reply:
Updated.